conviva/build.gradle

```diff
@@ -27,44 +28,36 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
-    namespace 'com.bitmovin.analytics.conviva'
+    buildFeatures {
+        buildConfig true
+    }
+    publishing {
+        singleVariant("release") {
+            withSourcesJar()
+        }
+    }
 }
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'androidx.appcompat:appcompat:1.6.1'
     testImplementation testingDependencies.junit
     androidTestImplementation testingDependencies.androidx_junit
     androidTestImplementation testingDependencies.androidx_espresso_core
-    api 'com.conviva.sdk:conviva-core-sdk:4.0.35'
+    api 'com.conviva.sdk:conviva-core-sdk:4.0.37'
     implementation bitmovinPlayerDependencies.bitmovinPlayer
-    implementation 'org.apache.commons:commons-text:1.7'
+    implementation 'org.apache.commons:commons-text:1.10.0'
 }
 
 publishing {
     publications {
-        aar(MavenPublication) {
+        release(MavenPublication) {
             groupId packageName
             version = libraryVersion
             artifactId project.getName()
 
-            // Tell maven to prepare the generated "*.aar" file for publishing
-            artifact("$buildDir/outputs/aar/${project.getName()}-release.aar")
-
-            pom.withXml {
-                def dependenciesNode = asNode().appendNode('dependencies')
-
-                //Defining configuration names from which dependencies will be taken (debugCompile or releaseCompile and compile)
-                def configurationNames = ['api']
-
-                configurationNames.each { configurationName ->
-                    configurations[configurationName].allDependencies.each {
-                        def dependencyNode = dependenciesNode.appendNode('dependency')
-                        dependencyNode.appendNode('groupId', it.group)
-                        dependencyNode.appendNode('artifactId', it.name)
-                        dependencyNode.appendNode('version', it.version)
-                    }
-                }
+            afterEvaluate {
+                from components.release
```

Comment:
Not true, I was comparing the wrong files. There are changes, that we should discuss on a video call.

Reply:
The POM file previously filtered out dependencies that our codebase actually needed (E.g. apache commons), which would cause problems for customers if they didn't define them on their own.
I removed all the not needed dependencies from the gradle setup itself, so the POM file is looking good now.

https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/59/commits/af789ba8256684d325e741230006910689aae02f